rpc/client.go

```diff
@@ -225,6 +232,14 @@ func uploadTestReport(signedURL string, data []byte, client *http.Client) error
 		return errors.Wrap(err, "report upload to given signed URL")
 	}
 
+	defer response.Body.Close()
+	grip.Debug(message.Fields{
+		"Status Code": response.StatusCode,
+	})
+	if response.StatusCode != http.StatusOK {
+		return errors.Errorf("Non-OK HTTP status. Status Code: %d. Status Text: %s", response.StatusCode, http.StatusText(response.StatusCode))
```

Comment:
Can you change this to `"failed to upload test report to the Data-Pipes service with response '%s'", response.Status`.